src/__tests__/message-id.integration.test.js

```diff
@@ -13,10 +13,12 @@ import { readFileSync } from 'fs';
 import expect from "expect";
 import nock from 'nock';
 import { sortBy } from 'lodash';
+import { replaceMessageIdsInObject } from "../services/transfer/transfer-out-util";
 
 describe('Replacement of message IDs', () => {
-  // ============ COMMON PROPERTIES ============
-  const gp2gpUrl = 'http://fake-gp2gpmessager-url';
+  const uuidRegexPattern = /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/
```

Comment:
`uuidRegexPattern `is being declared on both Line 19 & Line 90. Remove one or the other.

Reply:
Fixed